assets/component-facets.css

```diff
@@ -518,6 +518,21 @@ a.active-facets__button.focused .active-facets__button-inner,
   max-width: 37.5rem;
   display: flex;
   flex-direction: column;
+  --shadow-horizontal-offset: var(--popup-drawer-shadow-horizontal-offset);
+  --shadow-vertical-offset: var(--popup-drawer-shadow-vertical-offset);
+  --shadow-blur-radius: var(--popup-drawer-shadow-blur-radius);
+  --shadow-opacity: var(--popup-drawer-shadow-opacity);
+  border-bottom-left-radius: var(--popup-drawer-corner-radius);
+  border-top-left-radius: var(--popup-drawer-corner-radius);
+  border-color: rgba(var(--color-foreground), var(--popup-drawer-border-opacity));
+  border-style: solid;
+  border-width: 0 0 var(--popup-drawer-border-width);
+  filter: drop-shadow(
+    var(--shadow-horizontal-offset)
+    var(--shadow-vertical-offset)
+    var(--shadow-blur-radius)
+    rgba(var(--color-foreground), var(--shadow-opacity))
+  );
 }
```

Comment:
Should `.facets__display` get the same treatment ? Seems like a similar use case to the navigation submenus: 
![](https://screenshot.click/21-11-ejhln-feq5a.png)

Reply:
I believe they should 👍

Reply:
I agree 👍 Added. @melissaperreault let me know if you agree